ts/css_util.ts

```diff
@@ -15,65 +15,62 @@
  *  limitations under the License.
  */
 
-
 /**
- * @fileoverview Utility functions for adding CSS styles.
- *
+ * @file Utility functions for adding CSS styles.
  * @author v.sorge@mathjax.org (Volker Sorge)
  */
 
-import {HtmlClasses} from './html_classes.js';
-
+import { HtmlClasses } from './html_classes.js';
 
 /**
  * @namespace
  */
 export namespace CssStyles {
-
   /**
    * Creates CSS classname by prefixing a dot.
-   * @param {string} name The class name.
-   * @return {string} The context menu class name.
+   * @param name The class name.
+   * @returns The context menu class name.
    */
   function makeClass_(name: string): string {
     return '.' + (HtmlClasses[name] || name);
   }
 
-  let INFO_STYLES: {[id: string]: string} = {};
-  INFO_STYLES[makeClass_('INFOCLOSE')] = '{' +
-    '  top:.2em; right:.2em;' +
-    '}',
```

Comment:
These probably also wanted to be semicolons not commas, in order to avoid the extra parentheses that are being added.

Reply:
Yes.